guides/common/modules/proc_adding-server-as-a-dynamic-inventory-item.adoc

```diff
@@ -15,13 +15,13 @@ For more information about managing users, roles, and permission filters, see {A
 .Procedure
 
 . In the {awx} web UI, create a credential for your {Project}.
-For more information about creating credentials, see http://docs.ansible.com/ansible-tower/latest/html/userguide/credentials.html#add-a-new-credential[Add a New Credential] and http://docs.ansible.com/ansible-tower/latest/html/userguide/credentials.html#red-hat-satellite-6[{ProjectNameX} Credentials] in the _{awx} User Guide_.
+For more information about creating credentials, see http://docs.ansible.com/ansible-tower/latest/html/userguide/credentials.html#add-a-new-credential[Add a New Credential] and http://docs.ansible.com/ansible-tower/latest/html/userguide/credentials.html#red-hat-satellite-6[{ProjectName} Credentials] in the _{awx} User Guide_.
 +
 [[tabl-Managing_Hosts-Integrating_Satellite_and_Ansible_Tower-Satellite_Credentials]]
 .{Project} Credentials
 [cols="1a,2a"options="header"]
 |====
-|*Credential Type*: |*{ProjectNameX}*
+|*Credential Type*: |*{ProjectName}*
```

Comment:
This looks like the option within Ansible Tower - is that already modified to drop the version number?

Reply:
This may be replaced by mistake.

Reply:
I see, it's not in our interface. Reverting.